marketing/src/pages/about.tsx

```diff
@@ -26,11 +30,11 @@ const AboutPage: FC = () => {
 
           <meta
             name="og:description"
-            content="We urgently need to move towards a zero-carbon and nature positive future. We help companies evolve at the speed and scale needed for this, through technology, data, and advice on how to manage supply chain impacts."
+            content="We urgently need to move towards a zero-carbon and nature-positive future. We help companies evolve at the speed and scale needed for this, through technology, data, and advice on how to manage supply chain impacts."
           />
           <meta name="og:type" content="website" />
           <meta name="og:url" content="https://landgriffon.com/about" />
-          <meta name="og:image" content="https://landgriffon.com/images/og/OG-About.png" />
+          <meta name="og:image" content={`${domain}/images/og/OG-LandGriffon.png`} />
```

Comment:
@mariasola In order to test the previews before merging I created a Vercel project for the marketing site. And, for this, we require the use domain from ssr as you can see here. Could you apply this change to the rest of the pages?